store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9935,4 +9935,12 @@ TODO: delete them permanently from here
 <attr id="4006" name="zimbraServerExternalStoreConfig" type="string" cardinality="single" optionalIn="server" since="9.1.0">
   <desc>Server level external store config</desc>
 </attr>
+<attr id="4007" name="zimbraMultiReaderSMEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited"  since="9.1.0">
```

Comment:
It would be a better idea to prefix this attribute like below zimbraSMRuntimeSwitchEnabled, something like zimbraSMMultiReaderEnabled or zimbraStorageManagerMultiReaderEnabled

Reply:
make sense, done.